pkg/roachprod/prometheus/prometheus.go

```diff
@@ -390,7 +394,7 @@ func StopPrometheus(
 		"stop prometheus",
 		"sudo systemctl stop prometheus || echo 'Stopped prometheus'",
 	); err != nil {
-		return err
+		l.Printf("Failed to stop prometheus server")
```

Comment:
Also include the error? Looks like we're just eating the error here instead of bubbling it up. We're doing the same in the error handling codepaths above too, how come?

Reply:
Good catch!  Revised.

Reply:
I still see the old code without the error being bubbled up, did you mean to push a revision?